streams/next.json

```diff
@@ -524,104 +536,116 @@
+                            "image": "ami-00849c32686cc9d45"
                         },
                         "ap-northeast-2": {
-                            "release": "37.20221225.1.1",
-                            "image": "ami-040ee73ba69141b2e"
+                            "release": "37.20221225.1.2",
+                            "image": "ami-07ee953bfa3685a65"
                         },
                         "ap-northeast-3": {
-                            "release": "37.20221225.1.1",
-                            "image": "ami-0676c1913b6750cc0"
+                            "release": "37.20221225.1.2",
+                            "image": "ami-0f0f6a64182fe613b"
                         },
                         "ap-south-1": {
-                            "release": "37.20221225.1.1",
-                            "image": "ami-041b00cc70ce08896"
+                            "release": "37.20221225.1.2",
+                            "image": "ami-05e1a7044b6c1e537"
+                        },
+                        "ap-south-2": {
+                            "release": "37.20221225.1.2",
+                            "image": "ami-08188749948c82fc0"
                         },
                         "ap-southeast-1": {
-                            "release": "37.20221225.1.1",
-                            "image": "ami-01c57b59da2bcd0f2"
+                            "release": "37.20221225.1.2",
+                            "image": "ami-070844e2ba3079c9f"
                         },
                         "ap-southeast-2": {
-                            "release": "37.20221225.1.1",
-                            "image": "ami-06a7cc42312ec89be"
+                            "release": "37.20221225.1.2",
+                            "image": "ami-0647196a1471ef74d"
                         },
                         "ap-southeast-3": {
-                            "release": "37.20221225.1.1",
-                            "image": "ami-0711e410d2921f653"
+                            "release": "37.20221225.1.2",
+                            "image": "ami-0811e5aa61fe52469"
                         },
                         "ca-central-1": {
-                            "release": "37.20221225.1.1",
-                            "image": "ami-057b10b4881ffcdbf"
+                            "release": "37.20221225.1.2",
+                            "image": "ami-007e51673fe3ccd66"
                         },
                         "eu-central-1": {
-                            "release": "37.20221225.1.1",
-                            "image": "ami-01bdae15bac3c8acc"
+                            "release": "37.20221225.1.2",
+                            "image": "ami-0f26ecffa8577f3c2"
+                        },
+                        "eu-central-2": {
+                            "release": "37.20221225.1.2",
+                            "image": "ami-066b1dfe86cf21ffc"
```

Comment:
Wonder why the bot missed this addition and the one below for `eu-south-2`? cc @bgilbert

Reply:
Good catch!  It turns out that GitHub doesn't render more than 10 warnings: https://github.com/coreos/actions-lib/issues/10